core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/enclave_plan_instruction/enclave_plan_instruction_impl.go

```diff
@@ -0,0 +1,57 @@
+package enclave_plan_instruction
+
+import (
+	"encoding/json"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/enclave_plan_capabilities"
+	"github.com/kurtosis-tech/stacktrace"
+)
+
+type EnclavePlanInstructionImpl struct {
```

Comment:
nit: we usually don't like the `*Impl` name for struct. Why do you use it here? Is it because the `EnclavePlanInstruction` interface has only one implementation? If yes, then why do we even have an interface?